### DISCUSSION TITLE
How to limit the size of bounding boxes?

### DISCUSSION THREAD

**Discussion by Kuaranir** [ultralytics/yolov5#12979] [General]
I have trained YOLO5 on my custom dataset. Just one class.
My object is small, but sometimes YOLO shows me huge bounding box.
I want to exclude such mistakes and set _**max size of BB.**_
How can I do it?

**Comment by mayurkatre18** [comment]
After running of inference function for detecting object you can apply size filter for the BB in percentage of detection in image.
